flash/partition_info/pt.json

```diff
@@ -0,0 +1,47 @@
+{
+  "version": [1, 0],
+  "unpartitioned": {
+    "families": ["absolute"],
+    "permissions": {
+      "secure": "rw",
+      "nonsecure": "rw",
+      "bootloader": "rw"
+    }
+  },
+  "partitions": [
+    {
+      "name": "Firmware",
+      "id": 1,
+      "start": 0,
+      "size": "512K",
+      "families": ["rp2350-arm-ns", "rp2350-arm-s", "rp2350-riscv"],
```

Comment:
@will-v-pi Do these "family" identifiers also get stored in the partition table? Is it worth the example-code also printing them out?